Gemfile

```diff
@@ -55,28 +55,30 @@ gem 'nunes'
 gem 'opening_hours'
 gem 'postcode_anywhere-email_validation', '~> 0.2.0'
 gem 'psych', '>= 2.0.5' # https://www.ruby-lang.org/en/news/2014/03/29/heap-overflow-in-yaml-uri-escape-parsing-cve-2014-2525/
+gem 'rack', git: 'https://github.com/rails-lts/rack.git', branch: 'lts-1-6-stable'
 gem 'recaptcha', require: 'recaptcha/rails'
 gem 'redcarpet'
 gem 'rest-client', '~> 2.0'
 gem 'rouge'
-gem 'rubytree'
+gem 'rubytree', '~> 1'
 gem 'sass-rails'
 gem 'statsd-ruby'
 gem 'sucker_punch'
 gem 'turnout'
 gem 'whenever', require: false
+gem 'websocket-extensions', '>= 0.1.5'
 
 # MAS Gems
 # ========
 # Dependencies
 gem 'adal', git: 'git@github.com:moneyadviceservice/azure-activedirectory-library-for-ruby'
 gem 'cream', '2.1.8'
 gem 'dough-ruby', git: 'git@github.com:moneyadviceservice/dough.git', branch: '11691_DAL_Current-location-warning-message_v5.41', ref: 'a90d79d'
-gem 'mas-cms-client', '1.20.0'
+gem 'mas-cms-client', '1.20.1', git: 'git@github.com:moneyadviceservice/mas-cms-client.git', branch: 'TP-11734-upgrade-json'
 gem 'site_search', '0.3.0'
 # Tools
 gem 'action_plans', '~> 5.5.0'
-gem 'advice_plans', '~> 4.1.0'
+gem 'advice_plans', '~> 4.1.1', git: 'git@github.com:moneyadviceservice/advice_plans.git', branch: 'TP-11733-update-kaminari'
```

Comment:
Will the references to the `advice_plans` and `mas-cms-client` gems be updated to the new versions once they are merged?

Reply:
Yes, they will once this has had QA approval and I've merged the tools' feature branches.  I didn't want to do that prematurely.